src/Components/v2/Carousel.tsx

```diff
@@ -206,13 +206,23 @@ export class BaseCarousel extends React.Component<
           setCarouselRef(this.flickity)
         }
         this.flickity.on("select", this.handleSlideChange)
+        let startX
+        this.carouselRef.ontouchstart = e => {
+          startX = e.touches[0].clientX
+        }
+        this.carouselRef.ontouchmove = e => {
+          if (Math.abs(e.touches[0].clientX - startX) > 5 && e.cancelable) {
+            e.preventDefault()
+          }
+        }
```

Comment:
Can this be moved into a function so that it's clearer what's happening?

Reply:
👍